dns/pfSense-pkg-bind9/files/usr/local/pkg/bind_zones.xml

```diff
@@ -122,7 +122,7 @@
 			<type>select</type>
 			<options>
 				<option><name>Master</name><value>master</value><enablefields>description</enablefields></option>
-				<option><name>Slave</name><value>slave</value><enablefields>ttl</enablefields></option>
+				<option><name>Slave</name><value>slave</value><enablefields>tll</enablefields></option>
```

Comment:
Just this one part needs set back to TTL and then everything should be OK

Reply:
Actually the field is still called tll, I just fixed that line to match the existing typo.